docs/en_US/Tuner/BuiltinTuner.rst

```diff
@@ -49,7 +49,7 @@ Currently, we support the following algorithms:
 Usage of Built-in Tuners
```

Comment:
so we still keep the content below the table?

Reply:
The sentences do not appear anywhere else. They discuss in which case a tuner should be chosen.
So it should be a separate work to rearrange these contents.